src/solvers.jl

```diff
@@ -573,17 +573,18 @@ function vartypes_without_fixed(m::Model)
     return colCats
 end
 
-function collect_expr!(m, tmprow, terms::AffExpr)
+function collect_expr!(m, tmprow, terms::AffExpr, checkowned=true)
```

Comment:
True, that's a better naming.
When you have a constraint with variables from two models, calling `fillContrLHS!(I1, J1, V2, ..., m1, false)` and then `fillContrLHS!(I2, J2, V2, ..., m2, false)` will build two separate matrices. That is useful in StructJuMP for example. I will add some comments :)